static/js/settings_display.js

```diff
@@ -179,28 +179,18 @@ export async function report_emojiset_change(settings_panel) {
     }
 }
 
-export function update_page(settings_panel) {
-    const default_language_name = user_default_language_name;
-    const container = $(settings_panel.container);
-    const settings_object = settings_panel.settings_object;
-
-    // Boolean fields
-    container.find(".left_side_userlist").prop("checked", settings_object.left_side_userlist);
-    container.find(".translate_emoticons").prop("checked", settings_object.translate_emoticons);
-    container
-        .find(".escape_navigates_to_default_view")
-        .prop("checked", settings_object.escape_navigates_to_default_view);
-
-    // Enum/select fields
-    container.find(".default_language_name").text(default_language_name);
-    container
-        .find(".setting_twenty_four_hour_time")
-        .val(JSON.stringify(settings_object.twenty_four_hour_time));
-    container.find(".setting_color_scheme").val(JSON.stringify(settings_object.color_scheme));
-    container.find(".setting_default_view").val(settings_object.default_view);
+export function update_page(property) {
+    const container = $(user_settings_panel.container);
+    const value = user_settings[property];
 
     // TODO: Set emoji set selector here.
     // Longer term, we'll want to automate this function
```

Comment:
I think this comment probably can be deleted.  Though it's worth looking at whether we can make `settings_org.set_input_element_value` handle the `emojiset` type of element as well, the "automate this function" part is clearly addressed by calling that function rather than doing each element by hand.